tests/expectations/tests/class.rs

```diff
@@ -178,6 +194,7 @@ fn bindgen_test_layout_WithDtor() {
                 stringify ! ( b ) ));
 }
 #[repr(C)]
+#[derive(Copy)]
 pub struct IncompleteArrayNonCopiable {
```

Comment:
I think this should really be kept non-copiable though... That's a huge footgun, because incomplete arrays tend to be used to allocate to the end of the object, and copying it would lose the trailing information.